profiles/lib/relations.lua

```diff
@@ -17,4 +19,82 @@ function Relations.Merge(relations)
   return result
 end
 
+-- match ref values to relations data
+function Relations.MatchToRef(relations, ref)
```

Comment:
usually we use match_to_ref() instead of CamelCase for lua functions.

Reply:
Thanks, will fix it soon